helpers/localizeLabsElement.js

```diff
@@ -0,0 +1,11 @@
+import { LocalizeDynamicMixin } from '@brightspace-ui/core/mixins/localize-dynamic-mixin.js';
+
+export const LocalizeLabsElement = superclass => class extends LocalizeDynamicMixin(superclass) {
+
+	static get localizeConfig() {
+		return {
+			importFunc: async lang => (await import(`../src/lang/${lang}.js`)).default
+		};
+	}
+
+};
```

Comment:
This is probably fine for now. As more and more components get added here, it may make sense to split their localization out to avoid having the user download a bunch of translations for components that likely aren't rendered on the page.